config/routes.rb

```diff
@@ -2,6 +2,11 @@
 
 Rails.application.routes.draw do
   root 'home#index'
-
   post '/', to: 'home#search'
+
+  namespace :api do
+    namespace :v1 do
+      resources :developers, only: [:index]
```

Comment:
My explanation seems a bit vague.
I meant that I wanted to create a `show` method.
I corrected the exercise document.

You can keep this implementation.
Can you add `show`?

Reply:
@jp-ryuji ok (Y)